### DISCUSSION TITLE
Is there a callback function for successful verification？

### DISCUSSION THREAD

**Discussion by ghost** [nuxt-modules/turnstile#253] [Q&A] [answered]
### 📚 What are you trying to do?

I don't quite understand the document

**Comment by dargmuesli** [comment, marked as answer]
Yes, there is: https://github.com/nuxt-modules/turnstile/blob/main/src/runtime/types.ts#L25-L28

**Comment by dargmuesli** [comment]
@danielroe what do you think about enabling discussions on this repository to split bug reports and questions?

(migrated from issue)
